eventing/samples/writing-a-source/06-dispatching-events.md

```diff
@@ -0,0 +1 @@
+TODO
```

Comment:
Is this not critical for this guide? If it's not something you want to have done before this PR goes in, I'd vote for deleting this page until you're ready to add it. When we move over to a proper site (which is looking to be soon!) we won't want blank pages that are just TODOs.

Reply:
Good point, deleted this page until it can be written.